internal/repos/importer.go

```diff
@@ -169,3 +183,22 @@ func (i *importer) handleGitHubImport(ctx context.Context, imp db.ListRepoImport
 
 	return tx.Commit(ctx)
 }
+
+func handleDeletedRepos(ctx context.Context, tx pgx.Tx, i *importer, repos []*githubRepo, repoOwner string) error {
```

Comment:
```suggestion
func (i *importer) handleDeletedRepos(ctx context.Context, tx pgx.Tx, repos []*githubRepo, repoOwner string) error {
```

might as well make it a "private method" on the importer? One less arg to pass around